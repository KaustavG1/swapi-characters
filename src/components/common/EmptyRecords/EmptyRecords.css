.empty-record-message {
  font-size: 3rem;
}
