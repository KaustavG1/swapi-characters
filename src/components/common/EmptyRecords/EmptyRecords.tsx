import "./EmptyRecords.css";

function EmptyRecords() {
  return (
    <div className="blank-page">
      <div className="empty-record-message">No Records found.</div>
    </div>
  );
}

export default EmptyRecords;
