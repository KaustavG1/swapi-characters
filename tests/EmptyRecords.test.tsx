import { it, expect } from "vitest";
import { render, screen } from "@testing-library/react";
import EmptyRecords from "../src/components/common/EmptyRecords/EmptyRecords";

describe("EmptyRecords component", () => {
  it("renders 'EmptyRecords' component", () => {
    const { container } = render(<EmptyRecords />);
    const emptyRecords = screen.getByText(/No Records found./i);
    expect(emptyRecords).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
