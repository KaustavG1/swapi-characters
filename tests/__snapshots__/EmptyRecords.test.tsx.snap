// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`EmptyRecords component > renders 'EmptyRecords' component 1`] = `
<div>
  <div
    class="blank-page"
  >
    <div
      class="empty-record-message"
    >
      No Records found.
    </div>
  </div>
</div>
`;
